Cargo.toml

```diff
@@ -142,7 +142,7 @@ build-data = "0.1.3"
 bincode = "1.3.3"
 hex = "0.4.2"
 const_format = "0.2.30"
-num-bigint = "0.4"
+num-bigint = "0.4.4"
```

Comment:
I had to bump this because otherwise cargo couldn't reconcile the versions. It looks like the one needed by `light-poseidon` is compatible, but because it's a `dev-dependency`, by the time cargo got there it was locked at an earlier version.